meshroom/core/node.py

```diff
@@ -1220,6 +1220,7 @@ def has3DOutputAttribute(self):
 
     outputAttrEnabledChanged = Signal()
     hasImageOutput = Property(bool, hasImageOutputAttribute, notify=outputAttrEnabledChanged)
+    hasSequenceOutput = Property(bool, lambda self: any(attr.enabled and attr.isOutput and attr.desc.semantic == "sequence" for attr in self._attributes), notify=outputAttrEnabledChanged)
```

Comment:
Define a function (even if it's simple) for the getter instead of using a lambda in the property definition.